### PR TITLE
Popup Desktop and Mobile

### PR DESCRIPTION
**Pull request by touredri**
In this PR we have the following:

- Desktop popup window setup when see the project is clicked
- mobile popup window setup also
- Use of JavaScript to inflate HTML code into the document
- Use of JavaScript objects to store content for dynamic access

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -43,3 +41,261 @@ const toggleNav = () => {
...
+  {
+    title: 'Multi-Post Stories',
+    image: './images/desk2_SnapshootPortfolio.png',
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    technologies: ['html', 'css', 'bootstrap', 'Ruby'],
+    live: 'https://www.google.com/',
+    source: 'https://www.google.com/',
+    tag: ['Facebook', 'Full stack', '2015'],
+  },
+  {
```
Comment (ghost) [inline script.js:66]: - [ ] Your `see live` button on the pop-up appears to be outside the container. Kindly adjust to fit in within the container like in the original design

image

### Checkout the original,design

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Github workflow
✅ No linter error

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (touredri) [reply script.js:66]: I didn't see this before.
Right.

**Review by AbayomiOlaoye — APPROVED** [review]
Hi @touredri :wave: :wave:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Great implementation of reviewer's suggestions :heavy_check_mark: 
- Good UI implementation :heavy_check_mark: 
- Linter checks are passing :heavy_check_mark: 
- Professional ReadMe documentation :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**